jfa/list_issues.py

```diff
@@ -56,20 +56,32 @@ def get_issue_url(issue_key: str) -> str:
 # Convert the given dictionary representation of a Jira issue to a Alfred
 # feedback result dictionary
 def get_result_from_issue(issue: Issue) -> Result:
-    issue_type = issue["fields"]["issuetype"]["name"].lower()
-    status_name = issue["fields"]["status"]["name"]
+    fields = issue.get("fields", {})
+    issue_type = fields.get("issuetype", {}).get("name", "").lower()
+    issue_key = issue.get("key", "")
+    issue_summary = fields.get("summary", "")
+    status = fields.get("status", {}).get("name", "")
+    parent = fields.get("parent")
+    parent_key = parent.get("fields", {}).get("key") if parent else None
+    parent_summary = parent.get("fields", {}).get("summary") if parent else None
+
+    subtitle = f"{issue_key} • {status}"
+    if parent_summary:
+        subtitle += f" • {parent_summary}"
+
     return {
-        "title": issue["fields"]["summary"],
-        "subtitle": f"{issue['key']} ({status_name})",
-        "arg": issue["id"],
+        "title": issue_summary,
+        "subtitle": subtitle + " (view in Jira)",
```

Comment:
@jangelsb I removed the "(view in Jira)" suffix recently in 2ed6c418ea5ecebf1f5bf210e2f7f348bfc27851, so please make sure your fork is operating on the latest tip of the `main` branch.

Reply:
Hey @caleb531 - sorry for the latest response and update. I just got my branch rebased on yours and all up to date! 

I wasn't aware of how `TypedDict` worked - so I *think* I updated all that correctly - curious to see if we get a green build 🤞